dependencies/server-dependencies/pom.xml

```diff
@@ -169,7 +169,7 @@
         <dependency>
             <groupId>org.aerogear.unifiedpush</groupId>
             <artifactId>admin-ui</artifactId>
-            <version>1.2.4</version>
+            <version>1.3.0</version>
```

Comment:
once the JAR from the admin UI is release, this will be 1.2.6

until released, please use the snapshot.

We should never build non-snapshot things, outside of a release